src/viking.js

```diff
@@ -1,11 +1,98 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    this.attack = function attack () {
```

Comment:
Aunque esto te de bien, no es la forma correcta. Tenemos que crear un método donde se devuelve lo que nos pide: `this.strength` ó `this.health -= damage`

Reply:
Hecho.